### PR TITLE
Addresses several issues #gh-124, #gh-139, #gh-140 

### PR DESCRIPTION
**Pull request by bsbodden**
https://github.com/redis/redis-om-spring/issues/140
https://github.com/redis/redis-om-spring/issues/139
https://github.com/redis/redis-om-spring/issues/124

### CODE REVIEW

File: redis-om-spring/src/main/java/com/redis/om/spring/metamodel/SearchFieldAccessor.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+  public String getSearchAlias() {
+    return searchAlias;
+  }
+
+  public Class<?> getTargetClass() {
+    return targetClass;
+  }
+
+  public Class<?> getDeclaringClass() { return declaringClass; }
```
Comment (guyroyse) [inline redis-om-spring/src/main/java/com/redis/om/spring/metamodel/SearchFieldAccessor.java:33]: Consistency is the hobgoblin of a small mind.

File: redis-om-spring/src/main/java/com/redis/om/spring/repository/query/SearchLanguage.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  ITALIAN("italian"),
+  LITHUANIAN("lithuanian"),
+  NEPALI("nepali"),
+  NORWEGIAN("norwegian"),
+  PORTUGUESE("portuguese"),
+  ROMANIAN("romanian"),
+  RUSSIAN("russian"),
+  SERBIAN("serbian"),
+  SPANISH("spanish"),
+  SWEDISH("spanish"),
```
Comment (guyroyse) [inline redis-om-spring/src/main/java/com/redis/om/spring/repository/query/SearchLanguage.java:32]: ¿Come se dice Swedish en español?

File: redis-om-spring/src/test/java/com/redis/om/spring/annotations/document/DocumentLanguageTest.java [hunk 3]
```diff
@@ -25,26 +31,79 @@ class DocumentLanguageTest extends AbstractBaseDocumentTest {
...
+
+    MULTILINGUAL_SENTENCES.forEach((language, sentence) -> {
+      multiLingualDocRepository.save(MultiLingualDoc.of(language.getValue(), sentence));
+    });
   }
 
   @Test
   void testLanguage() {
-    SearchResult result = repo.findByBody("fusil");
+    SearchResult result = spanishDocRepository.findByBody("fusil");
```
Comment (guyroyse) [inline redis-om-spring/src/test/java/com/redis/om/spring/annotations/document/DocumentLanguageTest.java:65]: Bang! Bang!

Comment (guyroyse) [reply redis-om-spring/src/test/java/com/redis/om/spring/annotations/document/DocumentLanguageTest.java:65]: ¿Como se dice fusil en sueco?

**Review by guyroyse — CHANGES_REQUESTED** [review]

File: redis-om-spring/src/main/java/com/redis/om/spring/metamodel/SearchFieldAccessor.java (hunk 1 above)
Comment (bsbodden) [reply redis-om-spring/src/main/java/com/redis/om/spring/metamodel/SearchFieldAccessor.java:33]: New IDE doesn't autoformat on save :-( Need to find where to turn that on.

Comment (bsbodden) [reply redis-om-spring/src/main/java/com/redis/om/spring/metamodel/SearchFieldAccessor.java]: Fixed!

File: redis-om-spring/src/main/java/com/redis/om/spring/repository/query/SearchLanguage.java (hunk 2 above)
Comment (bsbodden) [reply redis-om-spring/src/main/java/com/redis/om/spring/repository/query/SearchLanguage.java]: Good catch!

**Review by guyroyse — APPROVED** [review]

**Comment by reemdiab** [comment]
Also is it possible to do full text search in stream ? as one of the filters ? 
Thanks

**Pull request merged**